// new way to create a function
// if the server response with error
Promise.catch((error) => {

});

